option optimize_for = LITE_RUNTIME;

message CommonSellAnswer
{
 required uint32 result = 1;
 repeated uint32 type = 2;
 repeated uint32 num = 3;
}
